data/flexibleForms/forms.types.ts

```diff
@@ -121,9 +126,16 @@ export interface Submission {
   residents: Resident[];
   workers: Worker[];
   editHistory: Revision[];
-  submissionState: 'In progress' | 'Approved' | 'Discarded' | 'Submitted';
+  submissionState:
+    | 'In progress'
+    | 'Approved'
+    | 'Discarded'
+    | 'Submitted'
+    | 'Panel Approved';
```

Comment:
It _might_ be nice to use an `enum` here rather than a union of strings:

```ts
enum SubmissionState {
    InProgress = "In progress",
    Approved = "Approved",
    Discarded = "Discarded",
    Submitted = "Submitted",
    PanelApproved = "Panel Approved"
};

submissionState: SubmissionState;
```